### PR TITLE
Update, simplify and modernise policy

### PR DESCRIPTION
**Pull request by justinmclean**

### CODE REVIEW

File: pages/policy/incubation.ad [hunk 1]
```diff
@@ -11,45 +11,51 @@ In October 2002 the Board of Directors of the Apache Software Foundation passed
...
 
-The Incubator was tasked with the following responsibilities:
-
-- the acceptance and oversight of new products submitted or proposed to become part of the Foundation;
-- providing guidance and ensuring that podlings under its purview develop products according to the Foundation's philosophy and guidelines for collaborative development; and
-- regularly evaluating products under its purview and making the de/guides/roles_and_responsibilities in each case of whether the product should be abandoned, continue to receive guidance and support, or proposed to the board for promotion to full project status as part of an existing or new Foundation PMC; and be it further.
+The Incubator has the following responsibilities:
+- Welcoming new Podlings to become part of the Apache Software Foundation.
+- Guiding Podlings to grow their communities according to the ASF's philosophy and guidelines for collaborative development (called the *Apache Way*).
+- Report monthly to the board on Podlings progress.
```
Comment (paulk-asert) [inline pages/policy/incubation.ad]: s/Report/Reporting/
s/on Podlings progress/on the progress of Podlings/

File: pages/policy/incubation.ad [hunk 2]
```diff
@@ -11,45 +11,51 @@ In October 2002 the Board of Directors of the Apache Software Foundation passed
...
+- Graduating Podlings to top level Apache Software Foundation projects.
+- When needed retiring Podlings.
```
Comment (paulk-asert) [inline pages/policy/incubation.ad]: Retiring Podlings when needed

**Comment by paulk-asert** [comment]
I added a couple of minor edit suggestions to make all sentences have a similar form.

File: pages/policy/incubation.ad (hunk 2 above)
Comment (justinmclean) [reply pages/policy/incubation.ad]: Thanks Paul I've made those changes.

File: pages/policy/incubation.ad [hunk 3]
```diff
@@ -72,25 +78,27 @@ project has yet to be fully endorsed by the ASF.
...
+See the guidelines for link:/guides/releasemanagement.html[Podling releases] for good practices on making releases.
+
+Podlings are not fully accepted as part of the Apache Software Foundation. Podlings MUST NOT perform any releases of software without the approval of the Incubator PMC. When a Podling decide it wants to make an release, the Podling MUST hold a vote on their public dev list. At least three +1 votes are required (see the http://www.apache.org/foundation/voting.html[Apache Voting Process] page) and more +1 votes than -1 votes. If the vote passes the Podling MUST send a summary of that vote to the Incubator's link:/howtoparticipate.html#Mailing+lists[general] list and request that the Incubator PMC approve the release. Three +1 Incubator PMC votes are required to approve a release. Below is an example showing how an incubating project managed this process:
```
Comment (paulk-asert) [inline pages/policy/incubation.ad]: s/accepted as _part of_ the/accepted as **official projects within** the/
s/When a Podling decide/When a Podling decide**s**/

File: pages/policy/incubation.ad [hunk 4]
```diff
@@ -72,25 +78,27 @@ project has yet to be fully endorsed by the ASF.
...
-Should the Incubator PMC, in accordance with these guidelines vote to approve the request, the Podling MAY perform the release under the following constraints:
+Should the Incubator PMC vote to approve a release, the Podling MAY make that release available to the public, only if:
 
-- the release archive MUST contain the word "incubating" in the filename; and
-- the release archive MUST contain an Incubation disclaimer (as described in the previous section), which SHOULD be placed in a DISCLAIMER file.
+- The release archive(s) MUST include the word "incubating" in the filename.
+- The release archive(s) MUST contain a disclaimer (see above), which SHOULD be placed in a DISCLAIMER file.
 
-Releases for *podling* MUST be distributed through *++http://www.apache.org/dist/incubator/++_podling_* In addition, the Podling MAY choose to distribute approved releases through other channels like the central Maven repository.
+Releases for *Podling* MUST be distributed through *++http://www.apache.org/dist/incubator/++_Podling_* Also the Podling MAY choose to distribute approved releases through other channels like the central Maven repository.
```
Comment (paulk-asert) [inline pages/policy/incubation.ad]: s/Releases for Podling/Releases for **the** Podling/

**Comment by paulk-asert** [comment]
Spotted a couple more very minor potential edits.

**Pull request merged**